ui/components/app/collectibles-items/collectibles-items.component.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+import Typography from '../../ui/typography/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+  JUSTIFY_CONTENT,
+  FLEX_DIRECTION,
+  ALIGN_ITEMS,
+  DISPLAY,
+  BLOCK_SIZES,
+  SIZES,
+  FLEX_WRAP,
+} from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../shared/constants/app';
+
+export default function CollectiblesItems({ onAddNFT, onRefreshList }) {
+  const t = useI18nContext();
+  const collections = {};
+  const defaultDropdownState = {};
+
+  Object.keys(collections).forEach((key) => {
+    defaultDropdownState[key] = true;
+  });
+
+  const [dropdownState, setDropdownState] = useState(defaultDropdownState);
+
+  return (
+    <div className="collectibles-items">
+      <Box padding={[4, 6, 4, 6]} flexDirection={FLEX_DIRECTION.COLUMN}>
+        <>
+          {Object.keys(collections).map((key, index) => {
+            const { icon, collectibles } = collections[key];
+            const isExpanded = dropdownState[key];
+
+            return (
+              <div key={`collection-${index}`}>
+                <Box
+                  marginTop={4}
+                  marginBottom={4}
+                  display={DISPLAY.FLEX}
+                  alignItems={ALIGN_ITEMS.CENTER}
+                  justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+                >
+                  <Box alignItems={ALIGN_ITEMS.CENTER}>
+                    <img width="28" src={icon} />
+                    <Typography
+                      color={COLORS.BLACK}
+                      variant={TYPOGRAPHY.H4}
+                      margin={[0, 0, 0, 2]}
+                    >
+                      {`${key} (${collectibles.length})`}
+                    </Typography>
+                  </Box>
+                  <Box alignItems={ALIGN_ITEMS.FLEX_END}>
+                    <i
+                      className={`fa fa-lg fa-chevron-${
+                        isExpanded ? 'down' : 'right'
+                      }`}
+                      onClick={() => {
+                        setDropdownState((_dropdownState) => ({
+                          ..._dropdownState,
+                          [key]: !isExpanded,
+                        }));
+                      }}
+                    />
+                  </Box>
+                </Box>
+                {isExpanded ? (
+                  <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP}>
+                    {collectibles.map((collectible, i) => {
+                      const width =
+                        getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
+                          ? BLOCK_SIZES.ONE_THIRD
+                          : BLOCK_SIZES.ONE_SIXTH;
```

Comment:
It should be possible to extract this piece of code into a constant and avoid calculating on each iteration
```
getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
                          ? BLOCK_SIZES.ONE_THIRD
                          : BLOCK_SIZES.ONE_SIXTH;
```